diff --git a/L2J_Server/build.xml b/L2J_Server/build.xml
index d9372816b..601e08678 100644
--- a/L2J_Server/build.xml
+++ b/L2J_Server/build.xml
@@ -59,7 +59,7 @@
 	</target>

 	<target name="compile" depends="version" description="Compile the source.">
-		<javac destdir="${build.classes}" optimize="on" debug="on" source="1.6" target="1.6" nowarn="off">
+		<javac destdir="${build.classes}" optimize="on" debug="on" source="1.7" target="1.7" nowarn="off">
 			<src path="${src}" />
 			<classpath refid="classpath" />
 		</javac>
diff --git a/L2J_Server/java/com/l2jserver/Config.java b/L2J_Server/java/com/l2jserver/Config.java
index df02356f3..ceb700639 100644
--- a/L2J_Server/java/com/l2jserver/Config.java
+++ b/L2J_Server/java/com/l2jserver/Config.java
@@ -202,6 +202,9 @@ public final class Config
 	public static boolean EXPERTISE_PENALTY;
 	public static boolean STORE_RECIPE_SHOPLIST;
 	public static boolean STORE_UI_SETTINGS;
+	public static boolean ALT_AUGMENTED_ITEM_TRADEABLE;
+	public static boolean ALT_AUGMENTED_ITEM_DROPABLE;
+	public static boolean ALT_AUGMENTED_ITEM_SELLABLE;

 	//--------------------------------------------------
 	// ClanHall Settings
@@ -1534,6 +1537,9 @@ public final class Config
 					EXPERTISE_PENALTY = Boolean.parseBoolean(Character.getProperty("ExpertisePenalty", "True"));
 					STORE_RECIPE_SHOPLIST = Boolean.parseBoolean(Character.getProperty("StoreRecipeShopList", "False"));
 					STORE_UI_SETTINGS = Boolean.parseBoolean(Character.getProperty("StoreCharUiSettings", "False"));
+					ALT_AUGMENTED_ITEM_TRADEABLE = Boolean.parseBoolean(Character.getProperty("AltAugmentedItemTradeable", "False"));
+					ALT_AUGMENTED_ITEM_DROPABLE = Boolean.parseBoolean(Character.getProperty("AltAugmentedItemDroppable", "False"));
+					ALT_AUGMENTED_ITEM_SELLABLE = Boolean.parseBoolean(Character.getProperty("AltAugmentedItemSellable", "False"));
 				}
 				catch (Exception e)
 				{
diff --git a/L2J_Server/java/com/l2jserver/gameserver/model/L2ItemInstance.java b/L2J_Server/java/com/l2jserver/gameserver/model/L2ItemInstance.java
index f07b6ff2d..047415860 100644
--- a/L2J_Server/java/com/l2jserver/gameserver/model/L2ItemInstance.java
+++ b/L2J_Server/java/com/l2jserver/gameserver/model/L2ItemInstance.java
@@ -660,7 +660,11 @@ public final class L2ItemInstance extends L2Object
 	 */
 	public boolean isDropable()
 	{
-		return isAugmented() ? false : _item.isDropable();
+		if (Config.ALT_AUGMENTED_ITEM_DROPABLE) {
+		  return _item.isDropable();
+		} else {
+			return isAugmented() ? false : _item.isDropable();
+		}
 	}

 	/**
@@ -678,16 +682,24 @@ public final class L2ItemInstance extends L2Object
 	 */
 	public boolean isTradeable()
 	{
-		return isAugmented() ? false : _item.isTradeable();
+		if (Config.ALT_AUGMENTED_ITEM_TRADEABLE) {
+		  return _item.isTradeable();
+		} else {
+			return isAugmented() ? false : _item.isTradeable();
+		}
 	}
-
+
 	/**
 	 * Returns if item is sellable
 	 * @return boolean
 	 */
 	public boolean isSellable()
 	{
-		return isAugmented() ? false : _item.isSellable();
+		if (Config.ALT_AUGMENTED_ITEM_SELLABLE) {
+		  return _item.isSellable();
+		} else {
+			return isAugmented() ? false : _item.isSellable();
+		}
 	}

 	/**
